packages/core/src/app/locale/translations/da.json

```diff
@@ -43,14 +44,14 @@
         },
         "billing": {
             "billing_address_heading": "Faktureringsadresse",
-            "billing_heading": "Fakturering",
+            "billing_heading": "fakturering",
```

Comment:
@bc-peng I remember you had some questions around this? Is that supposed to be capitalized?

Reply:
@funivan Could you please check is this better to be capitalized as you can see currently we are using `Fakturering` in production? This issue was first pointed out [here](https://github.com/bigcommerce/checkout-js/pull/883#discussion_r883218922).

Reply:
👋 I can change manually this string once in our Single Source of Truth. But unfortunately, there is no way to control if translators write words in a specific case.
__Changing manually:-_
pros - fast, time-consuming if we need to do this multiple times.
cons - we can have the same error tomorrow and it will be hard to spot it. 


Can we have a function that capitalizes the first letter and use it?

Reply:
I understand there are many ways to force capitalization on FE. However, I believe it is better to use the translation json files as the single source of truth to display any string on the checkout page.

Reply:
As an alternative mechanism, we can use a unit test to ensure that strings like this one are capitalised. 
@funivan What do you think?

Reply:
> However, I believe it is better to use the translation json files as the single source of truth to display any string on the checkout page

are there some patterns in what words we should capitalize first letter?

> As an alternative mechanism, we can use a unit test to ensure that strings like this one are capitalised.

Definitely, it will work to catch some wrong word formats. But what we will do next:
1. Change phrases in localization-dictionaries <- this is the source of truth for all translations that we have in the platform. 
2. Recreate the PR (this should be manually triggered)
This can cost time and effort every time we add a new language & phrases.